src/components/Rocket.js

```diff
@@ -1,26 +1,41 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
 
+import { changeRocketStatus } from '../redux/rockets/rocketsSlice';
 import './Rocket.css';
 
 const Rocket = (props) => {
   const {
-    id, name, description, images,
+    id, name, description, images, reserved,
   } = props;
 
   Rocket.propTypes = {
     id: PropTypes.number.isRequired,
     name: PropTypes.string.isRequired,
     description: PropTypes.string.isRequired,
     images: PropTypes.arrayOf(PropTypes.string).isRequired,
+    reserved: PropTypes.bool.isRequired,
+  };
+
+  const dispatch = useDispatch();
+
+  const handleChangeStatus = () => {
+    dispatch(changeRocketStatus(id));
   };
 
   return (
     <li id={id} className="rocket-container">
       <img src={images[0]} alt={name} style={{ height: 200 }} />
       <div className="rocket-info">
         <h3>{name}</h3>
-        <p>{description}</p>
+        <div className="rocket-description">
+          <p>
+            <span style={{ backgroundColor: 'blue' }}>{ reserved && 'Reserved' }</span>
```

Comment:
Instead of using a span and toggling whether there are words in it, you may use a React-Bootstrap badge and toggle whether it displays at all. I recommend this because they do look good and even if the color isn't exactly the same it is not hard to change.

See [here](https://react-bootstrap.github.io/getting-started/theming/) for information on customizing react-bootstrap.